Enterprise-CMCS/macpro-mdct-mcr
# [MDCT-2401] Disable form inputs for submitted MLR reports

### Description
Previously, submitted MLR reports could be modified by users, like MCPAR reports. This change removes this ability.

### Related ticket(s)


MDCT-2401

---
### How to test
1. Fill out a MLR report
2. Submit it 
3. Verify you can't edit form fields.

### Important updates
None


---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
